src/backend/routes/views/account/get/confirmPasswordReset.js

```diff
@@ -1,16 +1,40 @@
 exports = module.exports = function (req, res) {
     const locals = res.locals
 
+    const overallRes = res
+
     // locals.section is used to set the currently selected
     // item in the header navigation.
     locals.section = 'account'
 
     locals.formData = req.body || {}
 
-    const flash = null
-
     // Render the view
     locals.username = req.query.username
     locals.token = req.query.token
-    res.render('account/confirmPasswordReset', { flash })
+
+    // Ensure token and username are present
+    const validateParamPresence = (token, username) => {
```

Comment:
i know this works, but should we force to use "express-validator" for validations?
what do you think? 
```JavaScript
const { query, validationResult } = require('express-validator')
#....

 query('username')
        .isString().withMessage('Username missing')
        .trim()
        .notEmpty()
```

Reply:
Definitely cleaner and better option - will make the swap over.